code/index.html

```diff
@@ -3,28 +3,210 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <title>Design & Development</title>
+    <link href="https://fonts.googleapis.com/css?family=Lora|Roboto:400,700" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!--Header with hero-image-->
+    <header class="hero-image">
+      <h1>Design & Development. </h1>
+      <h3>Help you create the nicest websites you have ever seen!</h3>
+    </header>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <main> 
+    <!-- Signup form -->
+    <section class="form-container">
+      <div class="form-heading">  <!--Info text for form-->
+        <h2> Do you need help to set up a nice website?</h2>
+        <h4>Fill in the below form & we will contact you!</h4>
       </div>
+      <div class="form">
+        <form action="http://httpbin.org/anything" method="POST">
+           <!-- 2 x Text fields -->
+          <div class="form-div">  
+            <label for="fullName">Full name:</label>
+            <input 
+              id="fullName" 
+              type="text"
+              name="fullName"
+              required
+            />
+          </div>
+          <div class="form-div">
+            <label for="companyName">Company name:</label>
+            <input 
+              id="companyName" 
+              type="text"
+              name="companyName"
+            />
+          </div>
```

Comment:
You got some really nice structure with the indentation, this is super important and is much appreciated by the ones who are reading your code (- your future employer, perhaps?) 👏